xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -201,6 +213,9 @@ extern "C" {
     int (__cdecl* get_total_time)(const AddonInstance_InputStream* instance);
     int (__cdecl* get_time)(const AddonInstance_InputStream* instance);
 
+    // ITime
+    bool(__cdecl* get_times)(const AddonInstance_InputStream* instance, INPUTSTREAM_TIMES &times);
```

Comment:
I guess this should be a pointer, not a ref

Reply:
Does it make any difference?

Reply:
this is a C api, IMO refs are not supported

Reply:
ok, thx